src/lib/hotkey.model.ts

```diff
@@ -45,7 +47,7 @@ export class Hotkey {
      * @param allowIn     an array of tag names to allow this combo in ('INPUT', 'SELECT', and/or 'TEXTAREA')
      * @param persistent  if true, the binding is preserved upon route changes
      */
-    constructor(public combo: string | string[], public callback: (event: KeyboardEvent, combo: string) => ExtendedKeyboardEvent | boolean,
+    constructor(combo: string | string[], public callback: (event: KeyboardEvent, combo: string) => ExtendedKeyboardEvent | boolean,
```

Comment:
why it's moved out from here?

Reply:
Just so the cast to `Array<string>;` is not needed in the `formatted` getter as the constructor already ensures it's an array voa
```
        this.combo = (Array.isArray(combo) ? combo : [combo as string]);
```